### PR TITLE
Add cli

### PR DESCRIPTION
**Pull request by edenhaus**
Thanks for the help @rany2

### CODE REVIEW

**Comment by edenhaus** [comment]
@rany2  Can you please test it :) Then I will merge it today

File: deebotozmo/cli.py [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+    if cmd_args is None:
+        cmd_args = {}
+
+    util = CliUtil()
+    try:
+        await util.before(context.obj[DEVICE])
+        if isinstance(cmd_args, list):
+            await util.bot.execute_command(cmd(*cmd_args))
+        else:
+            await util.bot.execute_command(cmd(*cmd_args))
```
Comment (rany2) [inline deebotozmo/cli.py:179]: you mean `**cmd_args`

File: deebotozmo/cli.py [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+    debug: bool = False,
+    device: Optional[str] = None,
+) -> None:
+    """Create a click group for nesting subcommands."""
+    logging.basicConfig(format="%(name)-10s %(levelname)-8s %(message)s")
+    logging.root.setLevel(logging.DEBUG if debug else logging.WARNING)
+
+    # ensure that ctx.obj exists and is a dict (in case `cli()` is called
+    # by means other than the `if` block below)
+    if isinstance(ctx, click.Context):
```
Comment (rany2) [inline deebotozmo/cli.py:113]: I meant to make it `assert isinstance(ctx, click.Context)` but got lost with the PR

**Comment by edenhaus** [comment]
is everything working?

**Comment by rany2** [comment]
Yes, it works fine.

**Pull request merged**